### PR TITLE
Zally Web UI: Release Workflow

### LINKED ISSUE #462: Chore: release and publish task
Author: @rbarilani; Copied from the legacy Zally Web-UI repository.

Ideally we should have a npm script that release and publish a new version with one command.

```
$ npm run release [major|minor|path]
```

Pre-review:

- [ ] git checkout to new branch release/<next-version>
- [ ]  run tests
- [ ]  bump package.json version
- [ ]  build client assets
- [ ]  generate changelog
- [ ]  git add changes
- [ ]  git commit changes
- [ ]  git push changes
- [ ]  manually open a PR against master (or also this step can be automated?)

After review:

- [ ]  git add tag
- [ ]  git push tag
- [ ]  publish on npm

Ideally this task should run on a CI server but it should be possible to do that manually.

### PR DESCRIPTION
**Pull request by maxim-tschumak**
This PR includes:

* New release workflow documentation
* Package is moved to `@zalando` namespace
* Documentation (and repository) links have been fixed in package metadata

Closes #462.

### CODE REVIEW

File: web-ui/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "zally-web-ui",
+  "name": "@zalando/zally-web-ui",
```
Comment (netme) [inline web-ui/package.json]: As far as I remember, we also need to "register" an org in NPM (or maybe it's only a case for private NPM repos)

**Comment by netme** [comment]
👍

File: web-ui/package.json (hunk 1 above)
Comment (maxim-tschumak) [reply web-ui/package.json]: Zalando organization is already registered in NPM: https://www.npmjs.com/orgs/zalando.
Do you mean something else?

Comment (netme) [reply web-ui/package.json]: Ah, never mind, I guess it's only relevant for private NPM installations.

Comment (netme) [reply web-ui/package.json]: :shipit:

**Comment by maxim-tschumak** [comment]
:+1:

**Pull request merged**